src/cata_tiles.h

```diff
@@ -25,9 +25,37 @@ class JsonObject;
 struct visibility_variables;
 
 /** Structures */
+struct sprite_variation {
+    int weight;
+    std::vector<int> sprite_ids;
+
+    sprite_variation()
+    {
+        weight = 1;
```

Comment:
Why does the empty list start with a weight of 1? In line 723, you explicitly set the weight to -1, directly after construction a `sprite_variation`. This is very confusing. Apparently default  constructed objects of this type are not functional and need their `weight` member set to -1 manually, or not? If so, it should be set in the constructor (here).